plugins/pluginmanagers/lz-n.nix

```
@@ -190,10 +241,20 @@ nixvim.plugins.mkNeovimPlugin {
       };
     };
 
-  extraConfig = cfg: {
-    globals.lz_n = modules.mkAliasAndWrapDefsWithPriority id options.plugins.lz-n.settings;
-    plugins.lz-n.luaConfig.content = mkIf (cfg.plugins != [ ]) ''
-      require('lz.n').load( ${nixvim.toLuaObject cfg.plugins})
-    '';
+  extraConfig = cfg: opts: {
+    globals.lz_n = lib.modules.mkAliasAndWrapDefsWithPriority id opts.settings;
+    plugins.lz-n.luaConfig.content =
+      mkMerge [ (mkIf (cfg.plugins != [ ]) "require('lz.n').load(${toLuaObject cfg.plugins})") ]
+      ++ map (
+        {
+          plugin,
+          mode,
+          key,
+          action,
+          options,
+          ...
+        }:
+        "require('lz.n').keymap(${toLuaObject plugin}).set(${toLuaObject mode}, ${toLuaObject key}, ${toLuaObject action}, ${toLuaObject options})"
+      ) cfg.keymaps;
```

Needs parens to make the `++` apply to `mkMerge`'s arg, rather than concatenating the _result_ of calling `mkMerge`:

```suggestion
    mkMerge (
      [ (mkIf (cfg.plugins != [ ]) "require('lz.n').load(${toLuaObject cfg.plugins})") ]
      ++ map (
        {
          plugin,
          mode,
          key,
          action,
          options,
          ...
        }:
        "require('lz.n').keymap(${toLuaObject plugin}).set(${toLuaObject mode}, ${toLuaObject key}, ${toLuaObject action}, ${toLuaObject options})"
      ) cfg.keymaps
    );
```

---

oops 😅